README.md

```diff
@@ -400,6 +400,11 @@ The `camel` modifier will attach an event listener for the camel case equivalent
 
 > Note: `x-model` is smart enough to detect changes on text inputs, checkboxes, radio buttons, textareas, selects, and multiple selects. It should behave [how Vue would](https://vuejs.org/v2/guide/forms.html) in those scenarios.
 
+**`.number` modifier**
+**Example:** `<input x-model.number="age">`
+
+The `number` modifier will convert the input's value to a number.
```

Comment:
Shall we mention that the value is kept as it is if it's not a valid number?

Reply:
I was going to add that it's using `parseFloat` under the hood but decided against it, we're failing gracefully but I dunno how valuable it is that users know that (I think they would just assume it fails gracefully)

Reply:
we can write something which doesn't mention parseFloat (I copied Vue but we can use something like "If the value cannot be parsed as a valid number, the original value is returned.")
If we don't say anything, as a user I would assume that not valid number will be converted to 0 or NaN (which is what Alpine used to do before a PR changed that to match Vue)

Reply:
Updated to that 👍